### PR TITLE
add line about encoding to generated Python files

### PR DESCRIPTION
**Pull request by dirk-thomas**
Fixes ros/ros_comm#695.

@esteve @tfoote @wjwwood Please review.

### CODE REVIEW

File: src/genpy/generator.py [hunk 1]
```diff
@@ -896,6 +897,7 @@ def srv_generator(msg_context, spec, search_path):
     genmsg.msg_loader.load_depends(msg_context, spec, search_path)
     md5 = genmsg.compute_md5(msg_context, spec)
 
+    yield '# This Python file uses the following encoding: utf-8'
```
Comment (wjwwood) [inline src/genpy/generator.py]: This is unrelated to the pull request, but should this file also have a note about being auto-generated, as above:

https://github.com/ros/genpy/pull/41/files#diff-cda8e9d50594619a9ea61bdd3216b820R750

**Comment by wjwwood** [comment]
+1

File: src/genpy/generator.py (hunk 1 above)
Comment (dirk-thomas) [inline src/genpy/generator.py]: Actually this function doesn't need the change since it generates the message beforehand.

**Comment by tfoote** [comment]
+1

**Pull request merged**